tests/test_tree.py

```diff
@@ -882,3 +883,18 @@ def test_heap_float_values():
             assert root.min_leaf_depth == root_copy.min_leaf_depth
             assert root.min_node_value == root_copy.min_node_value + 0.1
             assert root.size == root_copy.size
+
+
+@pytest.mark.order14
+def test_get_parent_node():
```

Comment:
You are missing some test cases (e.g. when root is None).

Reply:
The test case root is None and node is None cases are added.